optparse.h

```diff
@@ -44,20 +44,28 @@
  * arguments to the end. This can be disabled by setting the `permute`
  * field to 0 after initialization.
  */
+
+#ifndef OPTPARSE_IMPLEMENTATION
+
 #ifndef OPTPARSE_H
 #define OPTPARSE_H
 
+#ifndef _WIN32
+#define TCHAR    char
+#define _TEXT(x) x
+#endif
+
```

Comment:
Are TCHAR and TEXT() macros still relevent? Could TCHAR be converted to `int` implicitly?
I'm haven't been using Windows for a long time. Please feal free to correct me.

https://docs.microsoft.com/en-us/windows/win32/learnwin32/working-with-strings